lib/utils/mu-sexp.cc

```diff
@@ -222,9 +223,15 @@ Sexp::to_json_string(Format fopts) const
 			auto it{list().begin()};
 			bool first{true};
 			while (it != list().end()) {
-				sstrm << (first ? "" : ",")  << quote(it->symbol().name) << ":";
+				const auto symbol = it->symbol();
+				sstrm << (first ? "" : ",")  << quote(symbol.name) << ":";
```

Comment:
This would be a bit more of a breaking change, but one could consider using `quote(symbol.asFieldName())` here to strip the colon and be closer to how the xml output looks (which does not have colons).